app/package-lock.json

```diff
@@ -0,0 +1,10372 @@
+{
```

Comment:
Defaultně používáme yarn, takže tenhle soubor by tam vůbec letět neměl. Jinak je to ok :)

Reply:
Upravila jsem. :)